packages/core/src/components/CodeSnippet/CodeSnippet.test.tsx

```diff
@@ -55,4 +55,14 @@ describe('<CodeSnippet />', () => {
     expect(queryByText(/2/)).toBeInTheDocument();
     expect(queryByText(/3/)).toBeInTheDocument();
   });
+
+  it('copy code using button', async () => {
+    document.execCommand = jest.fn();
+    const rendered = render(
+      wrapInTestApp(<CodeSnippet {...minProps} showCopyCodeButton />),
+    );
+    const button = rendered.getByTitle('Text copied to clipboard');
+    button.click();
```

Comment:
we're supposed to use `fireEvent.click(button)`, or even the newfangled `userEvent.click(button)`, aren't we?

Reply:
Copied that test from somewhere else, but I will take a look and may fix it at both places

Reply:
Switched to `fireEvent` as I couldn't find any uses of `userEvent`

Reply:
Makes sense! The userEvent library is pretty new and we haven't adopted it yet.